app.js

```diff
@@ -85,6 +87,8 @@ class App {
     this.infoDetailsClose = document.getElementById("info-details-close");
     this.infoDetailsDisplayed = false;
 
+    this.urlParams = new URLSearchParams(window.location.search);
```

Comment:
Would you mind moving the new url params into main.js so they're handled the same as the current url params (the pattern is that the url params are parsed in main.js and loaded into an "options" object which is passed to the App construtor).

Reply:
Much cleaner, done.